eng/templates/stages/release.yml

```diff
@@ -148,6 +150,18 @@ stages:
             draft='--draft'
           fi
 
+          git tag $releaseTag $dotnetDotnetCommit
+          
+          result=$(git push $publicVmrUrl refs/tags/$releaseTag 2>&1)
```

Comment:
How does authentication work here? I see that `GH_TOKEN` is set but is that actually consumed by `git`? I know that `gh` uses it but I'm not sure about `git`.

Reply:
I forgot about it honestly, thanks for pointing it out. Will fix

Reply:
Added the auth for the push